psi4/driver/driver_findif.py

```diff
@@ -850,63 +1021,70 @@ def init_string(data):
     return _process_hessian(H_pi, B_pi, massweighter, data["print_lvl"])
 
 
-def gradient_from_energies_geometries(molecule):
-    """
-    Generate geometries for a gradient by finite difference of energies.
-    
-    Parameters
-    ----------
-    molecule : qcdb.molecule or :py:class:`~psi4.core.Molecule`
-        The molecule to compute the gradient of.
-
-    Returns
-    -------
-    findifrec : dict
-        Dictionary of finite difference data, specified in _geom_generator docstring.
-
-    Notes
-    -----
-    Only symmetric displacements are necessary, so user specification of
-    symmetry is disabled.
-    """
-    return _geom_generator(molecule, -1, "1_0")
+########
 
+# This function is shifting `_energy_is_invariant` from driver to findif but isn't in final DDD form.
```

Comment:
The meaning of this string is lost on me. If this is a temporary function, I'll drop the matter.

Reply:
yes, it's presently ugly lines of code that are in the right place and will look more reasonable soon. suspended review requested.